statefun-flink/statefun-flink-common/src/main/java/org/apache/flink/statefun/flink/common/json/Selectors.java

```diff
@@ -89,6 +100,34 @@ public static int integerAt(JsonNode node, JsonPointer pointer) {
     return properties;
   }
 
+  public static Map<String, Long> longPropertiesAt(JsonNode node, JsonPointer pointer) {
```

Comment:
Can you add a test for that in `SelectorsTest` ?

Reply:
Done 👌